connector_magento/models/product_template/common.py

```diff
@@ -398,40 +398,18 @@ def escape(term):
             res = self._call('configurable-products/%s/children' % (escape(sku)), None)
             return res
 
-    def get_product_datas(self, data, id=None, saveOptions=True):
```

Comment:
could you explain what will be the replacement of this part ?
we need minimum datas while exporting templates and products.